include/deal.II/numerics/vector_tools.h

```diff
@@ -2516,6 +2516,82 @@ namespace VectorTools
    */
   template <int dim, class InVector, class OutVector, int spacedim>
   void
+  integrate_difference(
+    const Mapping<dim, spacedim> &                           mapping,
+    const DoFHandler<dim, spacedim> &                        dof,
+    const InVector &                                         fe_function,
+    const Function<spacedim, typename InVector::value_type> &exact_solution,
```

Comment:
This is backward compatible for the case `Function` is `double` and `InVector` is double. This is the most common case.